crates/biome_css_formatter/tests/specs/scss/expression/signed-numeric-operator.scss

```
@@ -26,3 +26,7 @@ $page-like-map:(
 (1 +2):foo,
 ($gap +1):bar,
 );
+
+$dimension-minus-left-space:10 -10px;
```

Stylelint (17.9.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)